e2e_test/source/basic/kafka.slt

```diff
@@ -363,6 +363,23 @@ create table s27 with (
   scan.startup.mode = 'earliest'
 ) FORMAT PLAIN ENCODE JSON (schema.location = 'file:///risingwave/json-complex-schema')
 
+# currently _rw_key can be set as primary key
+statement ok
+create source s28 (id bytea, PRIMARY KEY(_rw_key)) with (
```

Comment:
oh, sorry for my vague description, in RisingWave, creating source with the primary key is forbidden. I just talked about the table. I think you can remove this source test.

Reply:
got it